client/starwhale/api/_impl/data_store.py

```diff
@@ -4,11 +4,14 @@
 import json
 import atexit
 import base64
+import pickle
```

Comment:
use [dill](https://dill.readthedocs.io/en/latest/index.html#) to pickle python objects? 
In the starwhale, [PipelineHandler](https://github.com/star-whale/starwhale/blob/main/client/starwhale/api/_impl/model.py#L178) uses dill to pickle. dill can serialize more object types.

Reply:
Done